src/edge/tests/test_views.py

```diff
@@ -363,6 +363,27 @@ def test_add_annotation_on_forward_strand(self):
             "feature_base_last": 8,
         }])
 
+    def test_add_annotation_with_qualifiers(self):
+        data = dict(base_first=2, base_last=9, name='proC', type='promoter', strand=1,
+                    qualifiers=dict(gene="PROC"))
+        res = self.client.post(self.uri + 'annotations/', data=json.dumps(data),
+                               content_type='application/json')
+        self.assertEquals(res.status_code, 201)
+        self.assertEquals(json.loads(res.content), {})
+        res = self.client.get(self.uri + 'annotations/')
+        self.assertEquals(res.status_code, 200)
```

Comment:
minor but shall we start using self.assertEqual instead of self.assertEquals since py3 prefers self.assertEqual
https://stackoverflow.com/questions/930995/assertequals-vs-assertequal-in-python ```you should use whatever you like for Python 2.x, but tend toward assertEqual for Python 3.```